packages/create-schemas/src/utils.ts

```diff
@@ -87,3 +88,39 @@ export function formatBytes(bytes: number) {
 
     return `${gb.toFixed(2)} GB`;
 }
+
+export function getTsCompilerOptions(): ts.CompilerOptions {
```

Comment:
All of this is just to know if import paths should end with `".ts"`, `".js"` or `""`. 🤪